devtools/jenkins/conda_build.sh

```diff
@@ -8,14 +8,14 @@ export python_version=2.7
 conda build deepchem -c defaults -c rdkit -c omnia -c conda-forge
 export python_version=3.5
 conda build deepchem -c defaults -c rdkit -c omnia -c conda-forge
-export python_version=3.6
-conda build deepchem -c defaults -c rdkit -c omnia -c conda-forge
+#export python_version=3.6
+#conda build deepchem -c defaults -c rdkit -c omnia -c conda-forge
 
-export package_name=deepchem-gpu
-export tensorflow_enabled=tensorflow-gpu
-export python_version=2.7
-conda build deepchem -c defaults -c rdkit -c omnia -c conda-forge
-export python_version=3.5
-conda build deepchem -c defaults -c rdkit -c omnia -c conda-forge
-export python_version=3.6
-conda build deepchem -c defaults -c rdkit -c omnia -c conda-forge
+#export package_name=deepchem-gpu
```

Comment:
Not supporting GPUs out of the box seems pretty drastic.  If it's the only way to get it, we should consider downgrading to Tensorflow 1.4.  That's a much less serious change than dropping the GPU build.

Reply:
![selection_091](https://user-images.githubusercontent.com/1225605/37064364-bb69d0da-216a-11e8-9192-2f3e223e68e7.png)

So DeepChem GPU was downloaded 10 times as a package.  Not to say that 2.0.0 won't have larger user base etc.  The install from source still supports GPU.

We can change the build to not install tensorflow and then have the user manually pip install tensorflow afterwords?

Reply:
As a comparison the CPU only version was downloaded many more times.

![selection_092](https://user-images.githubusercontent.com/1225605/37064472-17721aea-216b-11e8-888a-1aa73195634a.png)

Reply:
![selection_093](https://user-images.githubusercontent.com/1225605/37064619-b49c90f2-216b-11e8-96d4-078d8894bc80.png)

I could add another line here for CPU/GPU with the final instruction to install tensorflow-gpu

Reply:
Also replace pip with docker and remove 3.6 as an option

Reply:
![selection_094](https://user-images.githubusercontent.com/1225605/37112484-95b1abd2-2210-11e8-9734-4b40ba1129dc.png)
New website up for PR looks like this for GPU enabled.

Reply:
I like the new installation instructions! I think it should be OK to remove deepchem-gpu with these in place.

I definitely don't want to us to downgrade TF, since eager mode is only enabled from 1.5 onwards and I think that eager integration is an important target for us to work towards.

Reply:
Well we still have to support the conda build for tensorflow-gpu (it is up and running [here](https://anaconda.org/deepchem/deepchem-gpu)  The conda install just has everything *except* tensorflow installed.